tools/oesign/oedump.c

```diff
@@ -178,6 +178,9 @@ void dump_enclave_properties(const oe_sgx_enclave_properties_t* props)
     bool debug = props->config.attributes & OE_SGX_FLAGS_DEBUG;
     printf("debug=%u\n", debug);
 
+    bool avx_supported = props->config.xfrm & SGX_XFRM_AVX;
+    printf("AVX Supported=%u\n", avx_supported);
```

Comment:
This is not exactly true, we don't rely on this bit to determine whether to load the signed enclave with or without the AVX flag anyway. If you want to dump information about this, I would just print the `%#x` of the of the whole `config.xfrm` value; there's nothing special about the AVX bit in this context.

Reply:
Good catch. I should remove these lines since I reverted the changes that I had made to oesign.c

Reply:
Good catch. Agree that this is not true since I reverted the changes to oesign.c. Will print the entire config.xfrm value.